### ISSUE TITLE
Build fails on mac

### ISSUE THREAD

**Issue by pfriesch** [Jonathan-LeRoux/lws#5]
```
cp -R ../lwslib .
cython --cplus lws.pyx -o lws.bycython.cpp
python setup.py build_ext -i
running build_ext
cythoning lws.pyx to lws.cpp
building 'lws' extension
creating build
creating build/temp.macosx-10.13-x86_64-2.7
creating build/temp.macosx-10.13-x86_64-2.7/lwslib
clang -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Ilwslib/ -I/Users/pfriesch/.pyenv/versions/virtenv/lib/python2.7/site-packages/numpy/core/include -I/Users/pfriesch/.pyenv/versions/2.7.6/include/python2.7 -c lws.cpp -o build/temp.macosx-10.13-x86_64-2.7/lws.o -O3
In file included from lws.cpp:581:
In file included from /Users/pfriesch/.pyenv/versions/virtenv/lib/python2.7/site-packages/numpy/core/include/numpy/arrayobject.h:4:
In file included from /Users/pfriesch/.pyenv/versions/virtenv/lib/python2.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:18:
In file included from /Users/pfriesch/.pyenv/versions/virtenv/lib/python2.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1816:
/Users/pfriesch/.pyenv/versions/virtenv/lib/python2.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:15:2: warning: "Using deprecated NumPy API, disable it by "
      "#defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-W#warnings]
#warning "Using deprecated NumPy API, disable it by " \
 ^
1 warning generated.
clang -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Ilwslib/ -I/Users/pfriesch/.pyenv/versions/virtenv/lib/python2.7/site-packages/numpy/core/include -I/Users/pfriesch/.pyenv/versions/2.7.6/include/python2.7 -c lwslib/lwslib.cpp -o build/temp.macosx-10.13-x86_64-2.7/lwslib/lwslib.o -O3
c++ -bundle -bundle_loader python.exe -L/usr/local/opt/readline/lib -L/usr/local/opt/readline/lib -L/usr/local/opt/openssl/lib -L/Users/pfriesch/.pyenv/versions/2.7.6/lib build/temp.macosx-10.13-x86_64-2.7/lws.o build/temp.macosx-10.13-x86_64-2.7/lwslib/lwslib.o -o /Users/pfriesch/GoogleDrive/projects/snips/ScalableFHVAE/datasets/librispeech_np_log_mag/fhvae_hs_e100_s5000_p10_a10.0_n5000/wav_tra/lws/python/lws.so
ld: file not found: python.exe
clang: error: linker command failed with exit code 1 (use -v to see invocation)
error: command 'c++' failed with exit status 1
make: *** [build] Error 1
```

**Comment by Jonathan-LeRoux** [comment]
Would this by any chance be related to [this bug](https://stackoverflow.com/questions/27259830/cant-build-python-gevent-on-mac-osx-10-10)?
It looks like you're using Python 2.7.6, and this is due to a Python bug that was fixed in 2.7.8.

**Comment by pfriesch** [comment]
That was it 👍Thank you! Works fine on 2.7.14
